### PR TITLE
175858970 release test

### PR DESCRIPTION
**Pull request by corydodt**

### CODE REVIEW

**Comment by corydodt** [comment]
Changes:

- adds getsentry/craft support (prepare & publish) including a bump-version.sh script, so we can stage and publish a release (in this case, just a tarball of the source)
- fixes django ugettext warnings
- removes empty "tests.py" files to prepare for app unit test coverage which will be centralized somewhere else
- tox changes and a github workflow to run the tests we have, on every push

File: .github/workflows/prepare_release.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  build:
+    runs-on: ubuntu-latest
+    name: "Build artifacts for the release"
+    needs: [prepare]
+    steps:
+      - id: set-version
+        run: |
+          echo 'RELEASE_VERSION=${{ github.event.inputs.version }}' >> $GITHUB_ENV;
+          echo 'RELEASE_PREFIX=jobbergate-api-${{ github.event.inputs.version }}' >> $GITHUB_ENV;
```
Comment (jamesbeedy) [inline .github/workflows/prepare_release.yaml:44]: 👍

File: .github/workflows/prepare_release.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      - uses: actions/checkout@v2
+        with:
+          ref: release/${{ env.RELEASE_VERSION }}
+
+      - id: set-git-user
+        run: |
+          git config user.name omnivector-qa
+          git config user.email admin@omnivector.solutions
+
+      - name: "tar it up"
```
Comment (jamesbeedy) [inline .github/workflows/prepare_release.yaml:55]: 👍

File: .github/workflows/test_on_push.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+name: 'Test'
+
+on: push
+
+jobs:
+  test:
```
Comment (jamesbeedy) [inline .github/workflows/test_on_push.yaml:6]: 👍

**Review by jamesbeedy — APPROVED** [review]
This all looks good to me. Let's do a quick sync on it before merge if you don't mind.

**Pull request merged**